### PR TITLE
Doc fixes and improvements

### PR DESCRIPTION
**Pull request by jonas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,17 @@ Full documentation [available here](https://diode.suzaku.io).
...
 
 In a Scala.js project the dependency looks like this.
 
-<pre><code class="lang-scala">"io.suzaku" %%% "diode" % "1.1.1"</code></pre>
+```scala
+"io.suzaku" %%% "diode" % "1.1.1"
+```
 
-<img align="right" src="doc/images/architecture.png">
+![Diode architecture](doc/images/architecture.png)
```
Comment (ochrons) [inline README.md]: This would lose the right alignment style which places the image to the right side of the text.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,7 @@
 >
 > a semiconductor device with two terminals, typically allowing the flow of current in one direction only
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ochrons/diode)
-[![Build Status](https://travis-ci.org/ochrons/diode.svg?branch=master)](https://travis-ci.org/ochrons/diode)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/suzaku-io/diode)
```
Comment (ochrons) [inline README.md]: There is a Travis CI build for the new repo as well at https://travis-ci.org/suzaku-io/diode

File: doc/usage/Handlers.md [hunk 3]
```diff
@@ -61,8 +61,6 @@ case class RootModel(tree: Tree)
 And some actions, too!
 
 ```scala
-trait Action
-
```
Comment (ochrons) [inline doc/usage/Handlers.md]: Yeah, makes sense now that there is a Diode defined `Action` available :)

File: README.md (hunk 2 above)
Comment (jonas) [inline README.md]: Ah, my bad, I probably made a typo when I looked for it. Fixed.

**Comment by jonas** [comment]
Reverted the image change, added back fixed Travis link and rebased.

**Pull request merged**